projects/packages/backup/src/js/components/Admin/header.js

```diff
@@ -0,0 +1,68 @@
+import { JetpackVaultPressBackupLogo } from '@automattic/jetpack-components';
+import { useSelect } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import useCapabilities from '../../hooks/useCapabilities';
+import { STORE_ID } from '../../store';
+import { useIsFullyConnected } from './hooks';
+
+const Header = () => {
+	const showActivateLicenseLink = useShowActivateLicenseLink();
+
+	return (
+		<div className="jetpack-admin-page__header">
+			<span className="jetpack-admin-page__logo">
+				<JetpackVaultPressBackupLogo />
+			</span>
+			{ showActivateLicenseLink && <ActivateLicenseLink /> }
+		</div>
+	);
+};
+
+const useShowActivateLicenseLink = () => {
+	const isFullyConnected = useIsFullyConnected();
+
+	const { capabilities, capabilitiesError, capabilitiesLoaded, hasBackupPlan } = useCapabilities();
+
+	return useMemo( () => {
+		const noBackupCapabilities =
+			! hasBackupPlan &&
+			capabilitiesLoaded &&
+			! capabilitiesError &&
+			Array.isArray( capabilities ) &&
+			capabilities.length === 0;
+
+		// This Boolean expression mirrors the logic found
+		// in the LoadedState component, in ./index.js,
+		// because these conditions are what triggers the pricing page to be displayed.
+		const shouldShow = isFullyConnected && noBackupCapabilities;
+
+		if ( ! shouldShow ) {
+			return null;
+		}
```

Comment:
```suggestion
		return shouldShow;
```

Should we return the value `shouldShow` instead of using this condition? Currently it is `undefined` if `shouldShow` is `true`.

Reply:
D'oh! That's what I get for going with a different idea in the middle of coding and not reading back through the whole thing one last time 🤦 Fixed (hopefully).